gitlabform/__init__.py

```diff
@@ -275,6 +279,15 @@ def _parse_args() -> Tuple:
             help="process only section with these names (comma-delimited)",
         )
 
+        sections_group.add_argument(
+            "-es",
+            "--exclude-sections",
+            dest="exclude_sections",
+            default="none",
+            type=str,
+            help="exclude sections with these names (comma-delimited). Warning: may result in odd functionality when enforcing different oders of operation.",
```

Comment:
Is the warning still valid? Also, still unsure if this is captured in the document.

Reply:
See above, perhaps the warning in CLI should point to README for full details 🤔

Reply:
I think reference to readme for full details is a good idea but that can be applicable for all the parameters (once we actually finish documenting them all). In that case, that reference will probably be at the beginning or end of the help output.

What do you think of rephrasing it like this?

```suggestion
            help="exclude sections with these names (comma-delimited). Warning: may result in failure or odd functionality when excluding sections that are dependent on by other sections.",
```